python/dgl/contrib/dis_sampling/receiver.py

```diff
@@ -0,0 +1,74 @@
+# This code implements a distributed dgl-subgraph receiver.
+from concurrent import futures
+
+import mxnet as mx
+import numpy as np
+
+import time
+from queue import Queue
+
+import grpc
+import sender
+import sampler_pb2
+import sampler_pb2_grpc
+
+_MAX_MESSAGE_LENGTH = 1024 ** 3  # Gigabyte
+
+class Receiver(sampler_pb2_grpc.SamplerServicer):
+    """ Reciever is called by trainer and it uses back-end thread
+        to receive sub-graph samples from remote sampler.
+    """
+    def __init__(self, port, buffer_size):
+    	self.port = port;
+    	self.queue = Queue(maxsize = buffer_size)
+
+    def SendSubGraph(self, request, context):
```

Comment:
why is this called SendSubGraph?

Reply:
It is a grpc handler, which will be invoked by sender. So in the sender side, it is called SendSubGraph.